.github/workflows/test.yml

```diff
@@ -30,14 +30,15 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
           cache: true
-
-      - name: go test
+      - uses: autero1/action-gotestsum@v2.0.0
+        with:
+          gotestsum_version: 1.11.0
+      - name: test enabled waf
         shell: bash
-        run: |
-          # Install gotestsum
-          env GOBIN=$PWD go install gotest.tools/gotestsum@latest
-          # Run the tests with gotestsum
-          env ${{ matrix.env }} CGO_ENABLED=${{ matrix.cgo_enabled }} ./gotestsum -- -v -count=10 -shuffle=on ./...
+        run: env ${{ matrix.env }} CGO_ENABLED=${{ matrix.cgo_enabled }} gotestsum -- -v -count=10 -shuffle=on ./...
+      - name: test disabled waf
+        shell: bash
+        run: env ${{ matrix.env }} CGO_ENABLED=${{ matrix.cgo_enabled }} gotestsum -- -v -tags datadog.no_waf -shuffle=on ./...
```

Comment:
I don't think this is necessary because this would be very tedious and we are htting our parralel job limit for the repo with only one PR already. But this is indeed not ideal, I will create another job running only what's necessary to test this